pyproject.toml

```
@@ -28,7 +28,7 @@ generate-setup-file = true
 script = "build.py"
 
 [tool.poetry.dependencies]
-python = ">=3.9,<3.13"
+python = "^3.9"
 numpy = "^1.23.0"
 scipy = "^1.8.1"
```

```suggestion
scipy = "^1.12.1"
```

It works for me if you bump SciPy too, and then remove it from `[tool.poetry.group.benchmark.dependencies]`

---

You are right, I made this changes and run poetry lock